fixtures/get-platform-proxy/tests/get-platform-proxy.env.test.ts

```diff
@@ -197,6 +205,55 @@ describe("getPlatformProxy - env", () => {
 		}
 	});
 
+	describe("DO warnings", () => {
+		let warn = {} as MockInstance<typeof console.warn>;
+		beforeEach(() => {
+			warn = vi.spyOn(console, "warn").mockImplementation(() => {});
+		});
+		afterEach(() => {
+			warn.mockRestore();
+		});
+
+		it("warns about internal DOs and doesn't crash", async () => {
+			await getPlatformProxy<Env>({
+				configPath: path.join(__dirname, "..", "wrangler_internal_do.jsonc"),
+			});
+			expect(warn).toMatchInlineSnapshot(`
+				[MockFunction warn] {
+				  "calls": [
+				    [
+				      "[33m▲ [43;33m[[43;30mWARNING[43;33m][0m [1m				You have defined bindings to the following internal Durable Objects:[0m
+
+				  				- {"class_name":"MyDurableObject","name":"MY_DURABLE_OBJECT"}
+				  				These will not work in local development, but they should work in production.
+				  
+				  				If you want to develop these locally, you can define your DO "externally" in another worker.
+				  				To do this, create another Worker and Wrangler config file. Export your DO from there.
+				  				Then, set the \`script_name\` field in your original DO binding to equal the \`name\` field from your new external DO config.
+				  
+				  				You will be able to develop this locally by running:
+				  				npx wrangler dev -c path/to/original/wrangler.json -c path/to/external-do/wrangler.json
```

Comment:
Not sure about this one but I think jsonc is what we recommend?
```suggestion
				  				npx wrangler dev -c path/to/original/wrangler.jsonc -c path/to/external-do/wrangler.jsonc
```